i18n-maven-plugin/pom.xml

```diff
@@ -35,16 +36,25 @@
   <packaging>maven-plugin</packaging>
   <name>ForgeRock I18N Maven Plugin</name>
   <description>A Maven plugin which generates I18N messages from property files</description>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>jp.openam.commons</groupId>
+        <artifactId>forgerock-bom</artifactId>
```

Comment:
Is it necessary to import BOM in this child project?

Reply:
Thank you for the review.
I deleted this part because it is unnecessary.